common/src/main/java/net/bettercombat/network/ServerNetwork.java

```diff
@@ -63,9 +65,17 @@ public static void initializeHandlers() {
             }
             final var packet = Packets.AttackAnimation.read(buf);
             final var forwardBuffer = new Packets.AttackAnimation(player.getId(), packet.animatedHand(), packet.animationName(), packet.length(), packet.upswing()).write();
+            try {
+                //send info back for Replaymod Compat
+                if (ServerPlayNetworking.canSend(player, Packets.AttackAnimation.ID)) {
+                    ServerPlayNetworking.send(player, Packets.AttackAnimation.ID, forwardBuffer);
+                }
+            } catch (Exception e){
+                e.printStackTrace();
+            }
```

Comment:
No. the way Replay mod works is by rolling back the packets send to the client by the server.
Checking if Replay Mod is installed on the server is kinda pointless, it would only fix the issue in singleplayer.

Reply:
we could, but i see no real purpose.
I changed the logic from the server refuses to send the client the packet about the animations
to the client ignoring the packet that regard themselfs so the animation is not played twice

Reply:
it is, sorry for the late response, i didnt change anyhting, that was in the PR from the start